spec/requests/breakers_integration_spec.rb

```diff
@@ -48,7 +48,7 @@
 
       stub_varx_request(:get, 'mhv-api/patient/v1/prescription/gethistoryrx', '{"message":"ack"}', status_code: 500)
       stub_varx_request(:get, 'mhv-api/patient/v1/prescription/getactiverx', '{"message":"ack"}', status_code: 500)
-      20.times do
+      80.times do
```

Comment:
Can you add a comment that the 80 is defined as the `breakers_error_threshold`

Reply:
@RachalCassity Do you know if that's true or not? The only reason I chose 80 was because thats what fixed the test through trial and error. We had a breakers_error_threshold of 50 before and this number was 20, so I don't know if there is a direct correlation. Let me know if that's the case and I will add the comment.

Reply:
Ohhh.. gotcha. I misunderstood.